scripts/ci/run-jenkins.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash -e
+
+TESTCMD=`dirname "${BASH_SOURCE[0]}"`/run-step.sh
+
+if [[ ${label} == debian-arm* ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; fi
+if [[ ${label} == 'osx-i386' ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no --build=i386-apple-darwin11.2.0"; fi
+if [[ ${label} == 'osx-amd64' ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no"; fi
+if [[ ${label} == 'w32' ]]; then PLATFORM=Win32; export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\Win32\\\bin\\\Release_SGen\\\mono-sgen.exe`";fi
+if [[ ${label} == 'w64' ]]; then PLATFORM=x64; export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\x64\\\bin\\\Release_SGen\\\mono-sgen.exe`"; fi
```

Comment:
Actually we do, since some of the jobs run with a local RabbitMQ -.-

Pull it out of here and into the Jenkins env var config?